### ISSUE TITLE
[TH] User Ratings - Quota not working

### ISSUE THREAD

**Issue by ShadowMCS** [Audentio/xfaddon-issues#49]
Quota does not work. They can only rep if it's set to unlimited If I have it set to say, 2, they are not allowed to give rep even if they haven't met the quota.

**Comment by ellisonpatterson** [comment]
When you say this do you mean that after 24 hours they are still unable to give more ratings?

**Comment by ShadowMCS** [comment]
No, they have not given ratings at ALL. The user I am testing it on has given 0 ratings ever.

**Comment by ellisonpatterson** [comment]
If they are assigned to more than one usergroup, are any of them set to 0?

**Comment by ShadowMCS** [comment]
No.

Groups of user:
https://gyazo.com/772210da819463658d12df77c6ad129d

Perms:
https://i.gyazo.com/0e0265d71c25b2d5b91292ac75fe8763

Error:
https://gyazo.com/9b9e9760d2474efd035c4a10bd63ef09

/ratings/ page (all reps given):
https://gyazo.com/8681d0728042f19891ae17775a03bf19

**Comment by ellisonpatterson** [comment]
I'll take a look and see what is causing this issue!

**Comment by ShadowMCS** [comment]
Thank you. Another admin from our site I believe contacted you, his name is Paxton. I believe he mentioned this before, I think he provided you a test account with the website (mcsell.org)

**Comment by ellisonpatterson** [comment]
Yeah however I couldn't seem to reproduce this. I may need FTP info to figure out what is going on exactly.

**Comment by ShadowMCS** [comment]
You couldn't reproduce it? Could you try again with the exact permissions and setup there?

About FTP, you'd need to get that from him, I don't have access to that.

**Comment by ShadowMCS** [comment]
Could you please take a look at the ticket I made?
https://www.themehouse.com/customer/tickets/8028

My other admin gave me access.
